extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.ViewPageTarget.js

```diff
@@ -59,7 +59,7 @@ ve.init.wikia.ViewPageTarget.static.toolbarGroups = [
 		type: 'list',
 		label: OO.ui.deferMsg( 'visualeditor-toolbar-insert' ),
 		indicator: 'down',
-		include: [ 'wikiaMediaInsert', 'wikiaSingleMedia', 'wikiaMapInsert', 'number', 'bullet', 'wikiaTemplateInsert', 'reference', 'referencesList', 'insertTable' ]
+		include: [ 'wikiaMediaInsert', 'wikiaSingleMedia', 'wikiaMapInsert', 'number', 'bullet', 'wikiaInfoboxInsert', 'wikiaTemplateInsert', 'reference', 'referencesList', 'insertTable' ]
```

Comment:
there is ticket for this, as well as for assigning a proper icon: https://wikia-inc.atlassian.net/browse/DAT-3120